regulations/uitests/definition_test.py

```diff
@@ -77,20 +78,19 @@ def test_definition(self):
         # go to 1005-1-a
         toc_toggle.click()
         WebDriverWait(self.driver, 10)
-        self.driver.execute_script(
-            "window.scrollTo(0, document.body.scrollHeight);")
+        self.driver.execute_script('window.scrollTo(0, 5);')
```

Comment:
Based on the next assertion, I'm not sure why we were scrolling lower in the document in the first place. We expect the wayfinding to show 1005.1 or 1005.1(a), but if we scroll to the bottom of the page, we're going to get the "wrong" wayfinding section (depending on screen height).